config/goerli-staging.json

```diff
@@ -53,6 +53,14 @@
       "address": "0x3b30d421a6dA95694EaaE09971424F15Eb375269",
       "name": "Endaoment Gallery for Good"
     },
+    {
+      "address": "0x2146a41f2c1432895d6d0adf9c60bf0a226bac0e",
+      "name": "Bright Moments - Flex"
+    },
+    {
+      "address": "0xa319c382a702682129fcbf55d514e61a16f97f9c",
+      "name": "Verse"
```

Comment:
nit: can we call this Verse - Flex?

Reply:
I was making this match the actual names of the contracts as configured at deployment, which is why BMG includes it (their preference) and Verse doesn't.

That said, can definitely use our own naming conventions here.. but wanted to flag this in case the additional context changed your opinion here on what makes sense.

WDYT?

Reply:
oh yeah, this is in the flex bucket already - resolving!